packages/db/src/schema.prisma

```diff
@@ -301,6 +337,10 @@ model Gloss {
   gloss      String?
   /// The approval state of the gloss.
   state      GlossState @default(UNAPPROVED)
+  /// The phrase the gloss applies to.
+  phrase     Phrase?    @relation(fields: [phraseId], references: [id])
```

Comment:
So glosses still apply to individual words, but they can also apply to an optional phrase?

Reply:
This is part of the expand/contract pattern that I laid out in the [ticket](https://github.com/globalbibletools/gbt/issues/376#issuecomment-2028517038). This PR is the first three steps, and the next PR will cover step 5, which we will release after we have completed steps 3 and 4 in production.